docs/content/dapp-development/mainnet-deployment.md

```diff
@@ -3,17 +3,15 @@ title: Mainnet Deployment Guidelines
 sidebar_title: 5. Mainnet Deployment
 ---
 
-> **Important**: The mainnet deployment process is changing! As soon as **[permissionless deployment](https://permissionless.onflow.org/)** is rolled out (ETA summer 2022), you will be able to deploy *new contracts* directly to mainnet without going through a review process. Furthermore, following the mainnet spork on June 15th, the network will already be permissionless for all *existing* contracts - this means you will be able to update your existing contracts without requiring any review. Please see [this page](https://docs.onflow.org/dapp-development/mainnet-deployment/#updatere-deploy-a-contract-on-mainnet-using-the-cli) or [this video](https://www.youtube.com/watch?v=tufIo8V_f2c) for information on how to update mainnet contracts after the spork. Please check [this page](https://permissionless.onflow.org/) for all the most up to date information.
+> **Note**: With the roll out of **[permissionless deployment](https://permissionless.onflow.org/)**, anyone can deploy and update contracts directly to the mainnet. [Smart contract reviews](https://flowsolutionseng.zendesk.com/hc/en-us/requests/new?ticket_form_id=360001936012&tc_360045236671=b5ba92954c2f2692b56099fe653ac92d35c1e6e6) (to discover any issues that may affect your users) are still provided upon request. However, they are not mandatory.
```

Comment:
For the first part, I would simplify to "Anyone can deploy and update contracts directly to mainnet"